pandas/tests/io/formats/test_info.py

```diff
@@ -417,31 +431,36 @@ def test_usage_via_getsizeof():
     assert abs(diff) < 100
 
 
-def test_info_memory_usage_qualified():
-
-    buf = StringIO()
-    df = DataFrame(1, columns=list("ab"), index=[1, 2, 3])
-    df.info(buf=buf)
-    assert "+" not in buf.getvalue()
-
-    buf = StringIO()
-    df = DataFrame(1, columns=list("ab"), index=list("ABC"))
-    df.info(buf=buf)
-    assert "+" in buf.getvalue()
-
-    buf = StringIO()
-    df = DataFrame(
-        1, columns=list("ab"), index=MultiIndex.from_product([range(3), range(3)])
-    )
-    df.info(buf=buf)
-    assert "+" not in buf.getvalue()
-
+@pytest.mark.parametrize(
+    "frame, plus",
+    [
+        (DataFrame(1, columns=list("ab"), index=[1, 2, 3]), False),
+        (DataFrame(1, columns=list("ab"), index=list("ABC")), True),
+        (
+            DataFrame(
+                1,
+                columns=list("ab"),
+                index=MultiIndex.from_product([range(3), range(3)]),
+            ),
+            False,
+        ),
+        (
+            DataFrame(
+                1,
+                columns=list("ab"),
+                index=MultiIndex.from_product([range(3), ["foo", "bar"]]),
+            ),
+            True,
+        ),
+    ],
+)
+def test_info_memory_usage_qualified(frame, plus):
```

Comment:
separating these i like, but im trying to push back against parametrization that creates DataFrame etc objects at test collection time.  it increases the memory footprint which is breaking the windows builds.

Reply:
Right, I remember your suggestion to reduce footprint.

Reply:
Reverted back.